Sources/TuistSigning/SigningCipher.swift

```diff
@@ -0,0 +1,151 @@
+import Basic
+import CryptoSwift
+import Foundation
+import TuistCore
+import TuistSupport
+
+enum SigningCipherError: FatalError, Equatable {
+    case failedToEncrypt
+    case failedToDecrypt(String)
+    case ivGenerationFailed(String)
+    case masterKeyNotFound(AbsolutePath)
+    case signingDirectoryNotFound(AbsolutePath)
+
+    var type: ErrorType { .abort }
+
+    var description: String {
+        switch self {
+        case .failedToEncrypt:
+            return "Unable to encrypt data"
+        case let .failedToDecrypt(reason):
+            return "Could not decrypt data: \(reason)"
+        case let .ivGenerationFailed(reason):
+            return "Generation of IV failed with error: \(reason)"
```

Comment:
Can users do something to mitigate this issue? If so, what about including those steps here?

Reply:
This error *should* never occur - and honestly, if it does, I am not sure how I'd fix it 😞

Reply:
That's fine! If this error is very unlikely, we don't need to put a lot of effort into figuring out how to handle it. If it turns out to be more likely, then it's worth the investigation. Thanks for clarifying it!